editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/index.tsx

```diff
@@ -0,0 +1,119 @@
+import Alert from "@mui/material/Alert";
+import Box from "@mui/material/Box";
+import Snackbar from "@mui/material/Snackbar";
+import { styled } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { TeamTheme } from "@opensystemslab/planx-core/types";
+import { FormikConfig } from "formik";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { useEffect, useState } from "react";
+import EditorRow from "ui/editor/EditorRow";
+
+import BoundaryForm from "./BoundaryForm";
+import ContactForm from "./ContactForm";
+import HomepagePlanningForm from "./HomepagePlanningForm";
+
+export const DesignPreview = styled(Box)(({ theme }) => ({
+  border: `2px solid ${theme.palette.border.input}`,
+  padding: theme.spacing(2),
+  boxShadow: "4px 4px 0px rgba(150, 150, 150, 0.5)",
+}));
+
+export const EXAMPLE_COLOUR = "#007078";
+
+export interface GeneralSettings {
+  boundaryUrl: string;
+  helpEmail: string;
+  helpPhone: string;
+  helpOpeningHours: string;
+  homepage: string;
+  isPlanningDataCollected: boolean;
+  portalName: string;
+  portalUrl: string;
+}
+
+export interface FormProps {
+  formikConfig: FormikConfig<GeneralSettings>;
+  onSuccess: () => void;
+}
+
+const GeneralSettings: React.FC = () => {
+  const [formikConfig, setFormikConfig] = useState<
+    FormikConfig<GeneralSettings> | undefined
+  >(undefined);
+
+  const initialValues = {
+    boundaryUrl: "",
+    helpEmail: "",
+    helpPhone: "",
+    helpOpeningHours: "",
+    homepage: "",
+    isPlanningDataCollected: true,
+    portalName: "",
+    portalUrl: "",
+  };
+
+  useEffect(() => {
+    const fetchTeam = async () => {
+      try {
+        setFormikConfig({
+          initialValues: initialValues,
+          onSubmit: () => {},
+          validateOnBlur: false,
+          validateOnChange: false,
+          enableReinitialize: true,
+        });
+      } catch (error) {
+        console.error("Error fetching team:", error);
+      }
+    };
+
+    fetchTeam();
+  }, []);
+
+  const [open, setOpen] = useState(true);
```

Comment:
```suggestion
  const [open, setOpen] = useState(false);
```

Currently this is open by default and appears when I load the page.


![image](https://github.com/theopensystemslab/planx-new/assets/20502206/067dad51-3642-4135-9c43-16ad9251cea6)